core/src/com/unciv/logic/city/CityResources.kt

```diff
@@ -33,25 +56,36 @@ object CityResources {
         return cityResources
     }
 
+    private fun addCityResourcesFromUniqueBuildings(city: City, cityResources: ResourceSupplyList, resourceModifer: HashMap<String, Float>) {
+        for (unique in city.getMatchingUniquesWithNonLocalEffects(UniqueType.ProvidesResources, StateForConditionals(city.civ, city))) { // E.G "Provides [1] [Iron]"
```

Comment:
This excludes follower religion and "In this city" uniques. Use `city.getMatchingLocalOnlyUniques`